### DISCUSSION TITLE
Some sites won't trigger extension dialog

### DISCUSSION THREAD

**Discussion by JoFrhwld** [coddingtonbear/obsidian-web#197] [Bug Report]
### Describe the bug

I've successfully installed the Local REST API and (I think) configured the chrome extension, but I'm finding the extension dialog triggers for *some* sites and not others. Interestingly, on a site that the dialog won't trigger, the extension badge *does* successfully display that the site is mentioned in one note.

## A working site:

1. Go to the obsidian-web wiki: https://github.com/coddingtonbear/obsidian-web/wiki
2. Click on the extension icon.
3. Dialog opens:

<img width="873" alt="Home_·_coddingtonbear_obsidian-web_Wiki" src="https://github.com/coddingtonbear/obsidian-web/assets/178755/a81c4ddd-2ace-4bac-96de-d8c4e26ea870">

## A non-working site

1. Go to this site: https://hoekhome.com/products/hoek-stack-desk-organizer
2. Click on the extension icon.
3. Dialog does not open, but the badge displays one mention.

<img width="727" alt="STACK___Modular_Desk_Organizer_–_Hoek_Home__Furniture__Simplified_" src="https://github.com/coddingtonbear/obsidian-web/assets/178755/0b6f7bfb-8340-4d69-aed6-5d46e71d948e">

### Expected Behavior

I expected the dialog to open both sites.

### Operating System

macos

### Browser

Chrome

### Browser Version

125.0.6422.142 (Official Build) (arm64)

**Comment by coddingtonbear** [comment]
I can confirm that that problem exists... I'm honestly not sure what's causing it on that particular site, though and might need somebody else to see if they can come up with a guess.
